api/v1beta1/csiaddonsnode_types.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2021.
```

Comment:
don't we have author for this one like https://github.com/csi-addons/kubernetes-csi-addons/pull/4/files#diff-ca1a7e8d22d8c8a4d2ef31d0898a86dd1fb989c820d3a7e88883f6d0e2d96cdbR2

Reply:
will add it.